app/src/main/java/com/futurice/freesound/network/api/model/Sound.kt

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright 2017 Futurice GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.futurice.freesound.network.api.model
+
+import android.os.Parcel
+import com.futurice.freesound.common.utils.*
+import com.squareup.moshi.Json
+import java.util.*
+
+/**
+ * Refer to: http://www.freesound.org/docs/api/resources_apiv2.html#sound-resources
+ */
+data class Sound(
```

Comment:
I would like to make the fields here non-nullable, perhaps with a sensible default, but the Freesound API is a little annoying in that you have to request specific fields you want in the response object, which would be great if we had dynamic typing, but not so good for us here. In any case, it's out of scope for this PR.